tests/OptionsTest.php

```diff
@@ -519,4 +519,34 @@ public function testReleaseOptionDefaultValueIsGotFromEnvironmentVariable(): voi
 
         $this->assertSame('0.0.1', (new Options())->getRelease());
     }
+
+    public function testErrorTypesIsDynamiclyReadingErrorReportingLevelWhenUnset(): void
+    {
+        $options = new Options(['error_types' => null]);
+
+        $errorReportingBeforeTest = error_reporting($currentErrorReporting = \E_NOTICE & \E_USER_NOTICE);
```

Comment:
Since we're messing with `error_reporting`, we should probably add a restore mechanism in `setup` and `teardown` of this test class.